### ISSUE TITLE
gammaQ() takes 1 positional argument but 2 were given

### ISSUE THREAD

**Issue by RJaBi** [gplepage/lsqfit#8]
The latest update b12d5d9788127c1bcbecea32b1b77d2576699422 breaks the gammaQ function.

Previously, the relevant code in `src/lsqfit/_scipy.py` was
` gammaQ = scipy.special.gammaincc `

and hence when called in `src/lsqfit/__init__.py` with the line
`self.Q = gammaQ(self.dof/2., self.chi2/2.)`

the gammaincc function from scipy was called correctly.

The latest change alters `src/lsqfit/_scipy.py` to
```
def gammaQ(x):
    from scipy.special import gammaincc
    return gammaincc(x)
```

and hence the call in `src/lsqfit/__init__.py` fails with the error message
`TypeError: gammaQ() takes 1 positional argument but 2 were given`

This error can be observed by trying to build the tests, i.e. make tests.
I am unsure if this error is also present when using the gsl library instead of scipy.

**Comment by gplepage** [comment]
Thanks for reporting the problem. It is fixed in the version I just uploaded (v12.0.1).

**Comment by RJaBi** [comment]
Thanks! 
Fixed by a63b61d3be2491e08dd298ac51477001cfc7e776
